src/SonarQube.Client/Api/DefaultConfiguration.cs

```diff
@@ -27,69 +38,41 @@ internal static class DefaultConfiguration
-            .RegisterRequest<IGetRulesRequest, V5_50.GetRulesRequest>("5.5")
-            .RegisterRequest<IDownloadStaticFile, V5_50.DownloadStaticFile>("5.5")
-            .RegisterRequest<IGetOrganizationsRequest, V6_20.GetOrganizationsRequest>("6.2")
-            .RegisterRequest<IGetProjectsRequest, V6_20.GetProjectsRequest>("6.2")
-            .RegisterRequest<IGetPluginsRequest, V6_30.GetPluginsRequest>("6.3")
-            .RegisterRequest<IGetPropertiesRequest, V6_30.GetPropertiesRequest>("6.3")
-            .RegisterRequest<IGetQualityProfileChangeLogRequest, V6_50.GetQualityProfileChangeLogRequest>("6.5")
-            .RegisterRequest<IGetQualityProfilesRequest, V6_50.GetQualityProfilesRequest>("6.5")
-            .RegisterRequest<IGetNotificationsRequest, V6_60.GetNotificationsRequest>("6.6")
-            .RegisterRequest<IGetRoslynExportProfileRequest, V6_60.GetRoslynExportProfileRequest>("6.6")
-            .RegisterRequest<IGetProjectBranchesRequest, V6_60.GetProjectBranchesRequest>("6.6")
-            .RegisterRequest<IGetOrganizationsRequest, V7_00.GetOrganizationsRequest>("7.0")
-            .RegisterRequest<IGetIssuesRequest, V7_20.GetIssuesRequestWrapper<V7_20.GetIssuesWithComponentSonarQubeRequest>>("7.2")
-            .RegisterRequest<IGetHotspotRequest, V8_6.GetHotspotRequest>("8.6")
-            .RegisterRequest<IGetExclusionsRequest, V7_20.GetExclusionsRequest>("7.2")
-            .RegisterRequest<IGetSonarLintEventStream, V9_4.GetSonarLintEventStream>("9.4")
+            .RegisterRequest<IGetRulesRequest, GetRulesRequest>("5.5")
+            .RegisterRequest<IGetPropertiesRequest, GetPropertiesRequest>("6.3")
+            .RegisterRequest<IGetQualityProfilesRequest, GetQualityProfilesRequest>("6.5")
+            .RegisterRequest<IGetNotificationsRequest, GetNotificationsRequest>("6.6")
+            .RegisterRequest<IGetProjectBranchesRequest, GetProjectBranchesRequest>("6.6")
+            .RegisterRequest<IGetIssuesRequest, GetIssuesRequestWrapper<GetIssuesWithComponentSonarQubeRequest>>("7.2")
+            .RegisterRequest<IGetExclusionsRequest, GetExclusionsRequest>("7.2")
+            .RegisterRequest<IGetSonarLintEventStream, GetSonarLintEventStream>("9.4")
             .RegisterRequest<ISearchHotspotRequest, V9_7.SearchHotspotRequest>("9.7")
-            .RegisterRequest<ISearchHotspotRequest, V10_2.SearchHotspotRequest>("10.2")
-            .RegisterRequest<IGetRulesRequest, V10_2.GetRulesWithCCTRequest>("10.2")
-            .RegisterRequest<ITransitionIssueRequest, V9_9.TransitionIssueRequestWithWontFix>("9.9")
-            .RegisterRequest<ITransitionIssueRequest, V10_4.TransitionIssueRequestWithAccept>("10.4")
-            .RegisterRequest<ICommentIssueRequest, V9_9.CommentIssueRequest>("9.9")
-            .RegisterRequest<ISearchFilesByNameRequest, V9_9.SearchFilesByNameRequest>("9.9");
+            .RegisterRequest<ISearchHotspotRequest, SearchHotspotRequest>("10.2")
```

Comment:
The usings cleanup would be nice to be reverted. Eg. In this case there are 2 classes with the same name and it can be easy to mix them up.
```suggestion
            .RegisterRequest<ISearchHotspotRequest, V10_2.SearchHotspotRequest>("10.2")
```

Reply:
😞